core/email_api.php

```diff
@@ -1301,6 +1301,77 @@ function email_bug_reminder( $p_recipients, $p_bug_id, $p_message ) {
 }
 
 /**
+ * Send a notification to user or set of users that were mentioned in an issue
+ * or an issue note.
+ *
+ * @param integer       $p_bug_id     Issue for which the reminder is sent.
+ * @param array         $p_mention_user_ids User id or list of user ids array.
+ * @param string        $p_message    Optional message to add to the e-mail.
+ * @param array         $p_removed_mention_user_ids  The users that were removed due to lack of access.
+ * @return array List of users ids to whom the reminder e-mail was actually sent
+ */
+function email_user_mention( $p_bug_id, $p_mention_user_ids, $p_message, $p_removed_mention_user_ids = array() ) {
+	if( OFF == config_get( 'enable_email_notification' ) ) {
```

Comment:
ok, so offtopic here: can we have 'enable_email_notification' overriden at user/project level? That is wicked...